cdash/nightly.sh

```diff
@@ -24,7 +24,8 @@ export CMAKE_PREFIX_PATH=$engpar:$kk:$oh:$pumipic:$CMAKE_PREFIX_PATH
 export MPICH_CXX=$root/kokkos/bin/nvcc_wrapper
 
 cd $root
-[ ! -d kokkos ] && git clone -b 4.0.01 git@github.com:kokkos/kokkos.git
+[ ! -d kokkos ] && git clone git@github.com:kokkos/kokkos.git
+cd kokkos && git checkout 4.1.00 && git pull && git -
```

Comment:
```suggestion
cd kokkos && git checkout 4.1.00 && git pull && cd -
```

Reply:
Thank you, I fixed this locally but I forgot to push.